src/components/Match/renderMatch.js

```diff
@@ -56,44 +56,80 @@ function generateGraphData(match) {
   return {};
 }
 
-function generateTeamfights(match) {
+function generateTeamfights({ players, teamfights = [] }) {
   const computeTfData = (tf) => {
     const newtf = {
       ...tf,
       deaths_pos: [],
+      radiant_networth_advantage_delta: 0,
```

Comment:
You may want to avoid using the term "networth" since gold and net worth are two separate stats.  This is actually gold (we don't track net worth right now)